src/esm_runscripts/filelists.py

```diff
@@ -911,10 +905,8 @@ def resolve_symlinks(config, file_source):
 
 
 def copy_files(config, filetypes, source, target):
-    if config["general"]["verbose"]:
-        print("\n::: Copying files", flush=True)
-        helpers.print_datetime(config)
-
+    logger.debug("\n::: Copying files")
+    helpers.print_datetime(config)
```

Comment:
outside verbose if statement?

Reply:
The `logger.info` call in `print_datetime` function was behind an if-statement referring to the variable `verbose_datetime_info` which does not exists anymore. So I deleted the if-statement and changed `logger.info` to `logger.debug` in the function `print_datetime`.